### PR TITLE
Add german version

### PR DESCRIPTION
**Pull request by faxemaxee**
* rough translation
* translator explanation regarding commit mesage language
* correct multiple grammar and spelling errors
* re-read the whole file and adjusted some semantic choices as well as squashing (hopefully all) typos and grammar errors
* update references in othe translation files

### CODE REVIEW

File: README_de-DE.md [hunk 1]
```diff
@@ -0,0 +1,432 @@
+# Commit Messages Guide
+
+Ein Leitfaden um die Wichtigkeit von Commit-Messages zu verstehen und diese richtig zu formulieren.
+
+Verstehe was ein Commit ist, warum es wichtig ist eine gute Commit-Message zu schreiben, Best-Practices und ein paar Tipps um eine saubere Commit-Historie zu planen oder vielleicht sogar nachträglich noch anzupassen.
+
+#### Hinweis des Übersetzers
+
+Auch wenn das eine Übersetzung darstellen soll, empfiehlt es sich, alle Commit-Messages auf Englisch zu schreiben,deshalb sind die Beispiele nicht übersetzt.
```
Comment (Ana06) [inline README_de-DE.md]: space after comma: `schreiben, deshalb`

File: README_de-DE.md [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+Adjust css
+```
+
+### Limitiere die Länge Commit-Message
+
+[Es wird empfohlen (Quelle in ENG)](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines) maximal 50 Zeichen für den Titel/Betreff und 72 Zeichen für die Nachricht zu verwenden.
+
+### Bleibe in einer Sprache
+
+Dies gilt vor allem für Entwickler für die Englisch nicht die Muttersprache ist. Logischerweise sollte man sich immer dem etablierten Standard anpassen – ganz besonders bei Repositories mit Entwicklern aus vielen verschiedenen Ländern.
```
Comment (Ana06) [inline README_de-DE.md:174]: This is not exactly translating the original one, I'd try to keep both versions the same.

**Review by Ana06 — CHANGES_REQUESTED** [review]

File: README_de-DE.md (hunk 1 above)
Comment (faxemaxee) [reply README_de-DE.md]: good catch, will fix

File: README_de-DE.md (hunk 2 above)
Comment (faxemaxee) [reply README_de-DE.md:174]: I see what you mean, although the version I forked already differs from the current HEAD. Feel free to suggest a translation and I'm happy to update mine.

Comment (Ana06) [reply README_de-DE.md:174]: I see, you were using the version before this commit https://github.com/RomuloOliveira/commit-messages-guide/commit/8c52dad1ad5b114e809a4f538203b57dc0777923

ok, then I would say it is fine. It is the problem of supporting several languages, when one is updated, the others need to be updated again. I think there are still some things to improve in the English version. So we can leave the German translation as it is, an in some time, synchronise it with the English one again. :wink:

Comment (faxemaxee) [reply README_de-DE.md:174]: Yes, I agree on that. ;)

File: README_de-DE.md (hunk 1 above)
Comment (Ana06) [reply README_de-DE.md]: Please add the space, so that we can :ship: it! :wink:

**Review by RomuloOliveira — DISMISSED** [review]
Hi, @faxemaxe, thank you for the German translation 🎉 

Since I do not speak or read German, I'll wait for @Ana06 review before merging it.

File: README_de-DE.md (hunk 1 above)
Comment (faxemaxee) [reply README_de-DE.md]: just did. happy 🚢-ing

**Review by Ana06 — APPROVED** [review]
My German is also not perfect, but this LGTM

Thanks @faxemaxe :bouquet:

**Comment by faxemaxee** [comment]
@RomuloOliveira You're very welcome. I'm excited to see where this will go and I will also keep an eye open on the English version.

@Ana06 thank you for the additional read-through.

**Pull request merged**